store/imagestore/store.go

```diff
@@ -58,6 +58,8 @@ const (
 
 	// how many backups to keep when migrating to new db version
 	backupsNumber = 5
+
+	Keylen = lenKey
```

Comment:
As per comment above, I don't think we should expose this to let consumers guess what they are handling.